llm4pddl/utils.py

```diff
@@ -356,3 +358,58 @@ def str_to_identifier(x: str) -> str:
         https://stackoverflow.com/questions/5297448
     """
     return hashlib.md5(x.encode('utf-8')).hexdigest()
+
+
+def get_visualization(input_path: str, output_dir: str) -> None:
+    """Creates visualization of planner accuracy across all environments."""
```

Comment:
clarify that the `input_path` should be to the `results_summary.csv` output by `analyze_results.py`